modules/install/pages/upgrade-strategy-for-features.adoc

```diff
@@ -24,15 +24,18 @@ If you plan to use the production release of Couchbase Analytics in version 6.0,
 | No
 | All Eventing nodes must be running version 6.0 to enable this feature.
 
-| Scorch indexing system (Full Text Search)
+| Scorch indexing system (Search Service)
 | Cluster
 | Offline/Online
 | Direct
 | Yes*
-| * FTS nodes running version 6.0 use the Scorch indexing system by default for new indexes.
-You can continue to use previous indexes in version 6.0, but they will not use Scorch.
+a| *After upgrading to Couchbase Server 6.0, existing Full Text Indexes will continue to use the Moss xref:fts:fts-creating-indexes.adoc#index-type[index type], while _new_ Full Text Indexes will use the Scorch index type by default.
```

Comment:
Question: what's the `a|` syntax here?

Reply:
@osfameron Good question. The `a` is the [cell specifier](https://docs.asciidoctor.org/asciidoc/latest/tables/add-cells-and-rows/#specifiers); more specifically the [style operator](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/). While I was drafting this change, I originally had a NOTE admonition in this cell. Since an admonition is a block, I needed to add an `a` in front of the cell separator for the block to render properly in the cell. However, I ended up removing the admonition but seemed to have left behind the cell specifier. It's harmless in this case and not really worth removing.

A note about adding the `a` to a cell: It truly turns the cell into a mini asciidoc document within the page. It allows you to treat the content like you would the any other content outside of the table, but causes some funny issues with certain asciidoc features. For example, if the cell contains a footnote reference, the footnote gets rendered at the bottom the cell, not the page.

Reply:
Ah! Thanks for that. Searching the docs for `a|` wasn't getting me anywhere...